src/main/java/ru/surpavel/churchshifts/entity/Day.java

```diff
@@ -0,0 +1,63 @@
+package ru.surpavel.churchshifts.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import org.hibernate.Hibernate;
+import org.hibernate.annotations.Type;
+
+import java.io.Serializable;
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Objects;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import javax.persistence.Temporal;
+import javax.persistence.TemporalType;
+
+@Getter
+@Setter
+@ToString
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@Table(name = "DAY")
+public class Day implements Serializable {
+	@Id
+	@Column(name = "ID")
+	@GeneratedValue(strategy = GenerationType.AUTO)
+	private Long id;
+
+	@Column(name = "DATE")
+	@Type(type = "org.jadira.usertype.dateandtime.joda.PersistentLocalDate")
+	private LocalDate date;
+
+	@OneToMany(targetEntity = Service.class, fetch = FetchType.LAZY)
+	@JoinColumn(name = "SERVICE", nullable = false)
+	@ToString.Exclude
+	private List<Service> services;
+
+	@Override
+	public boolean equals(Object o) {
+		if (this == o) return true;
```

Comment:
Надеюсь, это автоматически генерируемый код? Кстати, почему не lombok?

Reply:
перегенирировал. SonarLint ругается: Using @EqualsAndHashCode for JPA entities is not recommended. It can cause severe performance and memory consumption issues.